amethyst_core/src/lib.rs

```diff
@@ -8,23 +8,23 @@ compile_error!("the cfg flag \"no_threading\" is required when building for emsc
 
 pub use alga;
 pub use approx;
-pub use legion as ecs;
+pub use legion;
```

Comment:
We re-export here (`amethyst_core::legion`) and below (`amethyst_core::ecs::legion`), this could cause confusion at best. We should only re-export once.